source/draft_api.rst

```diff
@@ -561,7 +561,7 @@ Request body:
       "duration":12,
       "user": "example-2",
       "project": "ganet_web_manager",
```

Comment:
I'm curious. Is it intentional that this is spelled "ganet" (without an "i") rather than "ganeti"?